Vialer/AppDelegate.swift

```diff
@@ -216,6 +217,9 @@ extension AppDelegate {
             VialerGAITracker.acceptIncomingCallEvent()
         } else if notification.name == NSNotification.Name.CallKitProviderDelegateInboundCallRejected {
             VialerGAITracker.declineIncomingCallEvent()
+            if let mostRecentCallUnwrapped = self.mostRecentCall {
```

Comment:
You can also do something like this.
`vialerSIPLib.callManager.firstCallForAccount(vialerSIPLib.firstAccount())`
This will be nil or give the fist call for the account.
THen there is no need to set the most recent call.

Reply:
I am just getting always nil out of vialerSIPLib.callManager.firstCallForAccount(vialerSIPLib.firstAccount()) at that point of code. How can I get the most recent  call object with this approach?